Tweak interface colors for less obtrusive tabs

maxhawkins opened the pull request:
I find these minor design changes make Redcar more usable.

Making selected tabs grey instead of blue makes them less obtrusive and distracting.

Making unselected tabs darker makes them appear further behind the selected tab, focusing attention on the selected tab.

maxhawkins committed:
Tweak interface colors for less obtrusive tabs

Making selected tabs grey instead of blue makes them less obtrusive and distracting.

Making unselected tabs darker makes them appear further behind the selected tab, focusing attention on the selected tab.

maxhawkins committed:
Merge branch 'master' of git://github.com/redcar/redcar

maxhawkins committed:
Merge branch 'master' of git://github.com/redcar/redcar

danlucraft commented:
Basically, I like it. Make it configurable and it's an instant merge, otherwise I'll ask around and see what people think. Thanks.

maxhawkins committed:
Merge branch 'master' of git://github.com/redcar/redcar

maxhawkins committed:
Fix mistyped variable in vtab_label

Otherwise the program crashes when you try to draw a horizontal gradient.

maxhawkins committed:
Allow setting background on vtab_folder.

Gradients aren't implemented. The last color in the gradient is used as a solid background color.

maxhawkins committed:
Made tab colors customizable in preferences.

Selected tab color and unselected tab color are both customizable. The setting affects both the notebook and the treebook.

The color can either be entered as a hash table where the keys represent the positions of the colors: {0: "#E5E5E5", 100: "#D0D0D0"} or as a solid color: "#FEFE00"

Created gradient (adapter) class to encapsulate conversion from human-readable hash table format to SWT's difficult-to-understand format using two separate arrays.

Wrote spec to test Gradient class.

maxhawkins commented:
Here's a configurable version.

The colors also look a little different (and better.) I didn't realize I was using the gradient API incorrectly before.

danlucraft commented:
Merged this, thanks.

Can you send us to the mailing list the preference values needed to make it look like the old version, just in case anyone asks?

Thanks
